test/test_support.ts

```diff
@@ -37,7 +38,7 @@ function rootDir(): string {
  * tslib if we let it do its normal module resolution.  So instead we fix
  * the path here to the known path to the desired file.
  */
-const tslibPath = path.join(rootDir(), 'node_modules/tslib/tslib.d.ts');
+const tslibPath = path.join(rootDir(), '../npm/node_modules/tslib/tslib.d.ts');
```

Comment:
This won't sync nicely into g3. Why is this change needed?

Reply:
We can fix the g3 sync after, no big deal

Reply:
This change is needed because the old path assumed that node_modules was a directory in our repository (meaning you manually installed it before running bazel)
Now that bazel manages the node modules, it lays them out in an external repository called npm